[2025-05-14T15:35:15.906+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:30:00+00:00 [queued]>
[2025-05-14T15:35:15.916+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:30:00+00:00 [queued]>
[2025-05-14T15:35:15.917+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T15:35:15.918+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2025-05-14T15:35:15.919+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2025-05-14T15:35:15.935+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): Load_Data> on 2025-05-14 15:30:00+00:00
[2025-05-14T15:35:15.941+0000] {standard_task_runner.py:52} INFO - Started process 27250 to run task
[2025-05-14T15:35:15.944+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Sales_Performance_ETL', 'Load_Data', 'scheduled__2025-05-14T15:30:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_Hisham_Audiputra_Suharto_DAG.py', '--cfg-path', '/tmp/tmp7le64xg0', '--error-file', '/tmp/tmp0idptkve']
[2025-05-14T15:35:15.946+0000] {standard_task_runner.py:80} INFO - Job 119: Subtask Load_Data
[2025-05-14T15:35:16.018+0000] {task_command.py:371} INFO - Running <TaskInstance: Sales_Performance_ETL.Load_Data scheduled__2025-05-14T15:30:00+00:00 [running]> on host 2ca7303636ab
[2025-05-14T15:35:16.148+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Hisham
AIRFLOW_CTX_DAG_ID=Sales_Performance_ETL
AIRFLOW_CTX_TASK_ID=Load_Data
AIRFLOW_CTX_EXECUTION_DATE=2025-05-14T15:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-05-14T15:30:00+00:00
[2025-05-14T15:35:16.151+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2025-05-14T15:35:16.153+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'sudo -u *** python /opt/***/scripts/Load_Data.py']
[2025-05-14T15:35:16.164+0000] {subprocess.py:85} INFO - Output:
[2025-05-14T15:35:18.195+0000] {subprocess.py:92} INFO - WARNING: An illegal reflective access operation has occurred
[2025-05-14T15:35:18.197+0000] {subprocess.py:92} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/spark-unsafe_2.12-3.1.2.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2025-05-14T15:35:18.198+0000] {subprocess.py:92} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2025-05-14T15:35:18.199+0000] {subprocess.py:92} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2025-05-14T15:35:18.199+0000] {subprocess.py:92} INFO - WARNING: All illegal access operations will be denied in a future release
[2025-05-14T15:35:18.337+0000] {subprocess.py:92} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.7/site-packages/pyspark/jars/ivy-2.4.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-14T15:35:18.409+0000] {subprocess.py:92} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2025-05-14T15:35:18.410+0000] {subprocess.py:92} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2025-05-14T15:35:18.413+0000] {subprocess.py:92} INFO - org.postgresql#postgresql added as a dependency
[2025-05-14T15:35:18.413+0000] {subprocess.py:92} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-2e843e90-aba3-414f-858b-a53fd29a9ab3;1.0
[2025-05-14T15:35:18.414+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T15:35:18.501+0000] {subprocess.py:92} INFO - 	found org.postgresql#postgresql;42.6.0 in central
[2025-05-14T15:35:18.521+0000] {subprocess.py:92} INFO - 	found org.checkerframework#checker-qual;3.31.0 in central
[2025-05-14T15:35:18.538+0000] {subprocess.py:92} INFO - :: resolution report :: resolve 118ms :: artifacts dl 7ms
[2025-05-14T15:35:18.539+0000] {subprocess.py:92} INFO - 	:: modules in use:
[2025-05-14T15:35:18.540+0000] {subprocess.py:92} INFO - 	org.checkerframework#checker-qual;3.31.0 from central in [default]
[2025-05-14T15:35:18.541+0000] {subprocess.py:92} INFO - 	org.postgresql#postgresql;42.6.0 from central in [default]
[2025-05-14T15:35:18.542+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:35:18.543+0000] {subprocess.py:92} INFO - 	|                  |            modules            ||   artifacts   |
[2025-05-14T15:35:18.543+0000] {subprocess.py:92} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-05-14T15:35:18.544+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:35:18.544+0000] {subprocess.py:92} INFO - 	|      default     |   2   |   0   |   0   |   0   ||   2   |   0   |
[2025-05-14T15:35:18.545+0000] {subprocess.py:92} INFO - 	---------------------------------------------------------------------
[2025-05-14T15:35:18.545+0000] {subprocess.py:92} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-2e843e90-aba3-414f-858b-a53fd29a9ab3
[2025-05-14T15:35:18.545+0000] {subprocess.py:92} INFO - 	confs: [default]
[2025-05-14T15:35:18.546+0000] {subprocess.py:92} INFO - 	0 artifacts copied, 2 already retrieved (0kB/5ms)
[2025-05-14T15:35:18.726+0000] {subprocess.py:92} INFO - 25/05/14 15:35:18 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-14T15:35:18.951+0000] {subprocess.py:92} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2025-05-14T15:35:18.952+0000] {subprocess.py:92} INFO - Setting default log level to "WARN".
[2025-05-14T15:35:18.957+0000] {subprocess.py:92} INFO - To adjust logging level use sc.setLogLevel(newLevel). For SparkR, use setLogLevel(newLevel).
[2025-05-14T15:35:30.569+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2025-05-14T15:35:30.625+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Sales_Performance_ETL, task_id=Load_Data, execution_date=20250514T153000, start_date=20250514T153515, end_date=20250514T153530
[2025-05-14T15:35:30.657+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-05-14T15:35:30.696+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
